app/src/utils/paraspell.ts

```diff
@@ -44,7 +44,16 @@ export const createTx = async (
 export const getTokenSymbol = (sourceChain: TNodeDotKsmWithRelayChains, token: Token) => {
   const supportedAssets = getAllAssetsSymbols(sourceChain)
 
-  const tokenSymbol = supportedAssets.find(a => a.toLowerCase() === token.symbol.toLowerCase())
+  let tokenSymbol: string | undefined
+  if (sourceChain === 'Moonbeam') {
+    tokenSymbol = supportedAssets.find(a => {
+      const strippedAsset = a.toLowerCase().startsWith('xc')
+        ? a.toLowerCase().slice(2)
+        : a.toLowerCase()
```

Comment:
It would be nice to not repeat `toLowerCase` three times in a row 👍 

```suggestion
      const lowered = a.toLowerCase()
      const stripped = .startsWith('xc') ? lowered.slice(2) : lowered
     
```